drivers/hmis/lib/hmis_util/json_forms.rb

```diff
@@ -179,33 +222,98 @@ def load_definition(form_definition:, identifier:, role:)
+      record.title = title if title.present?
       record.save!
     end
 
-    # Load form definitions for editing and creating records
-    public def seed_record_form_definitions
-      record_forms.merge(service_forms).each do |identifier, form_definition|
+    public def ensure_system_instances_exist!
+      [
+        :project,
+        :organization,
+        :project_coc,
+        :funder,
+        :inventory,
+        :client,
+        :new_client_enrollment,
+        :enrollment,
+      ].each do |identifier|
         role = identifier.upcase.to_sym
-        role = :SERVICE if service_forms.key?(identifier)
-        raise "Unrecognized record form: #{identifier}" unless Hmis::Form::Definition::FORM_ROLES.key?(role)
+        raise "Unrecognized record form: #{identifier}" unless Hmis::Form::Definition::SYSTEM_FORM_ROLES.include?(role)
 
-        load_definition(
-          form_definition: form_definition,
-          identifier: identifier,
-          role: role,
-        )
+        default_instance = Hmis::Form::Instance.defaults.where(definition_identifier: identifier).first_or_create!
+        default_instance.update(system: true, active: true)
+        default_instance.touch
+      end
+    end
+
+    # Create some default HUD occurrence point instances
+    public def create_default_occurrence_point_instances!
+      # Move-in Date
+      unless Hmis::Form::Instance.where(definition_identifier: 'move_in_date').exists?
+        [3, 9, 10, 13].each do |ptype|
+          Hmis::Form::Instance.create!(
+            definition_identifier: 'move_in_date',
+            project_type: ptype,
+            data_collected_about: :HOH,
+            active: true,
+            system: false,
+          )
+        end
+      end
 
-        # Make this form the default instance  for this role. Don't do it for service forms since those
-        # need specialized instances based on service type.
-        unless service_forms.key?(identifier)
-          instance = Hmis::Form::Instance.find_or_create_by(entity_type: nil, entity_id: nil, definition_identifier: identifier)
-          instance.save!
+      # Date of Engagement
+      unless Hmis::Form::Instance.where(definition_identifier: 'date_of_engagement').exists?
+        # Note: spec has funder components too, but by default we just show it for all 3 project types.
+        [1, 4, 6].each do |ptype|
+          Hmis::Form::Instance.create!(
+            definition_identifier: 'date_of_engagement',
+            project_type: ptype,
+            data_collected_about: :HOH_AND_ADULTS,
+            active: true,
+            system: false,
+          )
         end
       end
 
-      puts "Saved definitions with identifiers: #{record_forms.keys.join(', ')}"
+      # PATH Status
+      return if Hmis::Form::Instance.where(definition_identifier: 'path_status').exists?
+
+      Hmis::Form::Instance.create!(
+        definition_identifier: 'path_status',
+        funder: 21,
```

Comment:
I think I'd rather see this get pulled from HudUtility2024 too, maybe `HudUtility2024.path_funder` or something.  All these magic numbers make me nervous.

Reply:
Updated